src/error_temp.c

```diff
@@ -0,0 +1,60 @@
+
+#include "mod_executor.h"
+
+/**
+ * @brief Print warning messages depending on 
+ * received indicator and trigger string.
+ * 
+ * @param indic		Indicator string for error message.
+ * @param trigger	Trigger string causing the error.
+ * @return t_err	ERR_HEREDOC_EOF, ERR_DIR, SUCCESS
+ */
+t_err	ft_print_warning(char *indic, char *trigger)
+{
+	if (!ft_strncmp(indic, "heredoc", 8))
+	{
+		ft_putstr_fd("minishell: warning: here-document at line 42 \
+	delimited by end-of-file (wanted `", 2);
+		ft_putstr_fd(trigger, 2);
+		ft_putendl_fd("')", 2);
+		return (ERR_HEREDOC_EOF);
+	}
+	else if (!ft_strncmp(indic, "dir", 4))
+	{
+		g_status = 126;
+		ft_putstr_fd("minishell: ", 2);
+		ft_putstr_fd(trigger, 2);
+		ft_putendl_fd(": Is a directory", 2);
+		return (ERR_DIR);
+	}
+	else
+		return (ft_print_warning2(indic, trigger));
+}
+
+/**
+ * @brief Continuation of ft_print_warning()
+ * 
+ * @param indic 	Indicator string for error message.
+ * @param trigger 	Trigger string causing the error.
+ * @return t_err 	ERR_DIR, SUCCESS
+ */
+t_err	ft_print_warning2(char *indic, char *trigger)
+{
+	if (!ft_strncmp(indic, "nodir", 6))
+	{
+		g_status = 127;
+		ft_putstr_fd("minishell: ", 2);
+		ft_putstr_fd(trigger, 2);
+		ft_putendl_fd(": No such file or directory", 2);
+		return (ERR_DIR);
+	}
+	if (!ft_strncmp(indic, "nocmd", 6))
+	{
+		g_status = 127;
+		ft_putstr_fd("minishell: ", 2);
+		ft_putstr_fd(trigger, 2);
+		ft_putendl_fd(": command not found", 2);
+		return (SUCCESS);
```

Comment:
Why SUCCESS and not ERR_UNKNOWN_CMD

Reply:
see above, basically when cmd is unknown, we require the pipeline to be continued and this is achieved with SUCCESS. May be changed in refactor

Reply:
As you write below you intentionally don't catch the error (executor.c:165). So i doesn't matter from a programming aspect but would make sense to be uniform